lib/foodcritic/command_line.rb

```diff
@@ -118,8 +204,18 @@ def version
     # @return [Boolean] True if the paths exist.
     def valid_paths?
       paths = options[:cookbook_paths] + options[:role_paths] +
-        options[:environment_paths]
-      paths.any? && paths.all? { |path| File.exist?(path) }
+              options[:environment_paths]
+      paths.flatten!
```

Comment:
I"m not sure I understand the change here... other than doing `all?` manually...